wordpress-mobile/WordPress-Android
# Total Likes and Comments cards

Fixes #16451 #16492

This PR adds new Total Likes card and Total Comments card to the stats screen. Since they are very similar cards, using the same endpoint, they're added in the same PR.

<img src="https://user-images.githubusercontent.com/2471769/167273745-d5c73484-2b14-4c9a-96ad-cd24108d8979.png" width=320>

Known issues:
- It's not added to default cards. Default cards will be set in another PR.
- Line chart will be added in another PR.

To test:
1. Go to App Settings (Tap on Avatar at the top right hand corner on My Site -> Find App Settings).
2. Select Debug Settings.
3. Find StatsRevampV2FeatureConfig under Features in development enable it and restart app.
4. Go to stats either using quick links or menu.
5. Ensure that it opens in Insights tab otherwise switch to Insights tab.
6. If the Total Likes and/or Total comments cards are not present, go to the stats management (either through ⚙️ menu at the top right hand corner or Add new stats card at the bottom) and add cards and save, then return to Insights.
7. Verify that the new Total Likes and Total Comments card are visible and matches the new design.
8. Check your total likes and total comments count from web and verify that the data in the new design is correct.

## Regression Notes
1. Potential unintended areas of impact
Only new cards were added to the list. It shouldn't affect any other view. 

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manually tested stats screens and views.

3. What automated tests I added (or what prevented me from doing so)
buildChange tests in `StatsUtilTest`, `TotalStatsMapperTest`, `TotalLikesUseCaseTest`, `TotalCommentsUseCaseTest`

PR submission checklist:
